shiny/render/_render.py

```diff
@@ -95,153 +69,33 @@ def text(
     --------
     ~shiny.ui.output_text
     """
-    return TextTransformer(_fn)
+
+    def default_ui(self, id: str, placeholder: bool | MISSING_TYPE = MISSING) -> Tag:
+        kwargs: dict[str, bool] = {}
+        if not isinstance(placeholder, MISSING_TYPE):
+            kwargs["placeholder"] = placeholder
+        return _ui.output_text_verbatim(id, **kwargs)
+
+    def __init__(self, fn: Optional[ValueFn[str]] = None, power: int = 1) -> None:
```

Comment:
`power` isn't supposed to be here, is it?

Reply:
:facepalm: No. Thank you! Removing